### ISSUE TITLE
Fonts not displaying

### ISSUE THREAD

**Issue by ronilaukkarinen** [alangrainger/share-note#127]
I have set my Obsidian Text font to Inter. It seems sharing feature has no fallback if custom font is set. Can you set it as system-ui somehow? Now it seems to revert to Times or similar.

**Comment by alangrainger** [comment]
Did you follow the steps in the troubleshooting link in the Issue creation screen?

https://docs.note.sx/Troubleshooting#force-a-full-re-upload-of-all-note-data

**Comment by ronilaukkarinen** [comment]
Yes I did follow the steps. Here is an example note: https://share.note.sx/qp73hbtv#ktSGrLOZZTHqP9zNXOlp01m6cHjaQSh5C/RSBDnVHko

**Comment by alangrainger** [comment]
Not sure sorry - here's an example of changing the Obsidian font, with a screenshot of the settings also:

https://share.note.sx/nv523t5v#oO8g9TkniYxyRY+Z7FS4cqOlGOGzBcVcKQYs2c43kVk

**Comment by alangrainger** [comment]
Are you using a custom theme? I am using the default theme, and this is what is set as fallback for <p>:

![image](https://github.com/user-attachments/assets/8e68e63a-fb23-4c57-9344-5520db7aa37f)

You can check the shared note here:

https://share.note.sx/nv523t5v#oO8g9TkniYxyRY+Z7FS4cqOlGOGzBcVcKQYs2c43kVk

I realised thanks to your issue that default fonts aren't uploaded since they aren't part of the theme. I might see if I can upload the Obsidian set font also.

**Comment by ronilaukkarinen** [comment]
I'm using the default theme with [obsidian-minimal-settings]([obsidian-minimal-settings](https://github.com/kepano/obsidian-minimal-settings)), some CSS variables changed in snippets (no fonts though) and that's it.

> However I realised thanks to your issue that default fonts aren't uploaded since they aren't part of the theme. I might see if I can upload the Obsidian set font also.

Yes, I have only set my fonts via Obsidian Core settings, Text is set as Inter. That's it. I think there is no way to set fallback fonts there? So what do you suggest I'll try?

I think you can reproduce this like this:

1. Go to **Appearance > Font > Text Font > Manage**
2. Change to some font you have downloaded from Internet, any font that is not a system font
3. Try sharing a link

For me it seems to have `font-family: Inter` (I've set the Text Font as Inter) so the browser falls back to Times... or somethin like that. In the example my monospace font shows because it is installed on my system.

<img width="736" alt="image" src="https://github.com/user-attachments/assets/c9d58727-18a8-4af0-8db7-9090bae5972c">

**Comment by alangrainger** [comment]
> I think you can reproduce this like this

That's exactly how I shared the last note, those exact 3 steps. 

So maybe you can try in a Sandbox vault? You can see that my default Obsidian installation does send a fallback font option.

**Comment by ronilaukkarinen** [comment]
I figured it out. I have multiple fonts set up. If the font contains spaces, it breaks it and reverts to browser default.

![image](https://github.com/user-attachments/assets/5fcbbdd3-6da1-4b01-bc42-302813d233b6)

If I set Inter, Alliance No 2, SN Pro, it doesn't work. The setting prints out it like this to the shared page:

```css
--font-text-override: Inter, Alliance No .2, SN Pro;
```

![image](https://github.com/user-attachments/assets/d6155e65-bfc0-4054-9f44-de00c2954b33)

If I only set Inter, it works:

```css
--font-text-override: Inter;
```

https://share.note.sx/1zd49xzt#vGmIa7o+ZgmyQHmM8eOuh/ug8eP7eQvgQxoMlKoMdUs

![image](https://github.com/user-attachments/assets/92b8da01-feae-44b4-9b80-0d130c08f067)

**Comment by alangrainger** [comment]
Based on your syntax highlighting here, it might actually be the font with the dot `.` in it:

```css
--font-text-override: Inter, Alliance No .2, SN Pro;
```

If you open up the Developer console in Obsidian and look at the CSS for those elements, does Obsidian quote the font strings? I'm getting the CSS directly from Obsidian's DOM, so it will be an exact replica. However I'm not sure why Obsidian can correctly display the unquoted fonts.

**Comment by ronilaukkarinen** [comment]
My examples were directly from devtools. So no, Obsidian doesn't quote fonts. That is seemingly the culprit here.
